docs/coinjoin/joinmarket.md

```diff
@@ -0,0 +1,60 @@
+# JoinMarket
+
+## Overview
+
+JoinMarket is software to create a special kind of bitcoin transaction called a ["CoinJoin"](https://en.bitcoin.it/wiki/CoinJoin). Its aim is to improve the confidentiality and privacy of bitcoin transactions. JoinMarket differs from other popular CoinJoin implementations in a few ways.
+
+The most important differences are:
+
+1) Users can choose to act as a market "taker" and initiate a CoinJoin transaction on demand, or they can act as a market "maker" (and potentially earn fees) by leaving a JoinMarket wallet connected to the network for others to CoinJoin with.
+2) JoinMarket has no central coordinator.
+
+JoinMarket was created by Chris Belcher in 2015, and is currently maintained by Adam Gibson [here](https://github.com/joinmarket-clientserver/joinmarket).
```

Comment:
URL is wrong, should be https://github.com/JoinMarket-Org/joinmarket-clientserver.

Reply:
Thanks! Done